lib/factory-helper/timestamps.rb

```diff
@@ -0,0 +1,33 @@
+module FactoryHelper
+  class Timestamps
+    class << self
+
+      def between(from=::Time.at(0), to=::Time.now)
+        random_time(from, to)
+      end
+
+      def forward(seconds= 60)
+        from = ::Time.now
```

Comment:
The entropy leak in the determinism test failure is because `Time.now` will not return the same result on successive runs with the same seed.  If `Time.now` is used, the user should pass it as an argument.  This way, the gem always returns the same value on the same input with the same seed.